scripts/docker-run.sh

```diff
@@ -19,10 +19,11 @@ fi
 docker run -d \
   --name $CONTAINER_NAME $VOLUME_MOUNT \
   -p ${PGPORT}:5432 \
+  -e POSTGRES_PASSWORD=password \
```

Comment:
I don't think hardcoding a password here is the correct solution

Reply:
Changed to parameter